application/controllers/QuestionEditorController.php

```diff
@@ -590,7 +590,7 @@ public function actionGetQuestionPermissions($iQuestionId = null)
             "editorpreset" => App()->session['htmleditormode'],
             "script" =>
                 Permission::model()->hasSurveyPermission($oQuestion->sid, 'surveycontent', 'update')
-                && SettingsUser::getUserSetting('showScriptEdit', App()->user->id) == 1,
+                && SettingsUser::getUserSettingValue('showScriptEdit', App()->user->id) == 1,
```

Comment:
`SettingsUser::getUserSettingValue('showScriptEdit', App()->user->id)` only please :) `== 1` not needed

Or if you really want ti be clear `boolval(SettingsUser::getUserSettingValue('showScriptEdit', App()->user->id))`
https://www.php.net/manual/en/function.boolval.php

Reply:
ok,thank you